dev/archery/archery/integration/datagen.py

```diff
@@ -1492,21 +1492,25 @@ def _temp_path():
 
         generate_primitive_large_offsets_case([17, 20])
         .skip_category('Go')
-        .skip_category('JS'),
+        .skip_category('JS')
+        .skip_category('Rust'),
 
         generate_null_case([10, 0])
+        .skip_category('Rust')
```

Comment:
Didn't I see a patch merge recently that added null support? If you rebase, can you remove this skip?